### PR TITLE
Restrictions input changed 

### PR DESCRIPTION
**Pull request by karthickbharathi1**

### CODE REVIEW

File: frontend/src/components/Request.jsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export default function Request() {
     phoneNumber: '',
     pickupDate: '',
     selectedPersonalCareProducts: [],
+    allergies: '',
```
Comment (jibranabsarulislam) [inline frontend/src/components/Request.jsx:42]:
```suggestion
    restrictions: '',
```

File: frontend/src/components/Request.jsx [hunk 2]
```diff
@@ -170,6 +171,17 @@ export default function Request() {
           />
         </Input>
 
+        <Input label="Allergies">
```
Comment (jibranabsarulislam) [inline frontend/src/components/Request.jsx:174]:
```suggestion
        <Input label="Restrictions (dietary, allergies, etc.)">
```

File: frontend/src/components/Request.jsx [hunk 3]
```diff
@@ -170,6 +171,17 @@ export default function Request() {
...
+          <input
+            type="text"
+            name="allergies"
```
Comment (jibranabsarulislam) [inline frontend/src/components/Request.jsx:177]:
```suggestion
            name="restrictions"
```

File: frontend/src/components/Request.jsx [hunk 4]
```diff
@@ -170,6 +171,17 @@ export default function Request() {
...
+            placeholder="Allergies"
```
Comment (jibranabsarulislam) [inline frontend/src/components/Request.jsx:178]:
```suggestion
            placeholder="peanuts, vegan, gluten, etc."
```

File: frontend/src/components/Request.jsx [hunk 5]
```diff
@@ -170,6 +171,17 @@ export default function Request() {
...
+            value={formData.allergies}
```
Comment (jibranabsarulislam) [inline frontend/src/components/Request.jsx:179]:
```suggestion
            value={formData.restrictions}
```

File: frontend/src/components/staff/RequestList.jsx [hunk 6]
```diff
@@ -89,6 +89,7 @@ export default function RequestList({ refresh, onRefresh }) {
             <p>Email: {selectedRequest?.email}</p>
             <p>Phone Number: {selectedRequest?.phoneNumber}</p>
             <p>Pickup Date: {selectedRequest?.pickupDate}</p>
+            <p>Allergies: {selectedRequest?.allergies}</p>
```
Comment (jibranabsarulislam) [inline frontend/src/components/staff/RequestList.jsx:92]:
```suggestion
            <p>Restrictions: {selectedRequest?.restrictions}</p>
```

File: frontend/src/components/staff/RequestList.jsx [hunk 7]
```diff
@@ -138,6 +139,7 @@ export default function RequestList({ refresh, onRefresh }) {
                   <p>Email: {req.email}</p>
                   <p>Phone Number: {req.phoneNumber}</p>
                   <p>Pickup Date: {req.pickupDate}</p>
+                  <p>Allergies: {req.allergies}</p>
```
Comment (jibranabsarulislam) [inline frontend/src/components/staff/RequestList.jsx:142]:
```suggestion
                  <p>Restrictions: {req.restrictions}</p>
```

**Review by jibranabsarulislam — CHANGES_REQUESTED** [review]
looks great! I've suggested changes from allergies to restrictions, as it would better encompass the information we're looking to get and store from the user. thanks again @karthickbharathi1

**Comment by karthickbharathi1** [comment]
I had now committed the suggested changes.

**Review by jibranabsarulislam — APPROVED** [review]
looks great, thanks a ton @karthickbharathi1

**Comment by jibranabsarulislam** [comment]
@all-contributors please add @karthickbharathi1 for code

**Pull request merged**